src/services/BookstoreAPI.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-undef */
```

Comment:
- [ ] Just as I pointed out above, I'll suggest you remove these linters disable rule and try to fix the errors. :+1: :small_airplane: :smile:  also remove the one in the last line of this file. :+1: